.github/workflows/auto-dev-release.yml

```
@@ -1,174 +1,179 @@
-            echo "❌ package.json missing 'version' field"
-            exit 1
-          fi
-          
-          echo "✅ package.json validation passed"
-
-      - name: Install dependencies
-        if: steps.check_changes.outputs.skip_release == 'false'
-        run: npm ci
-
-      - name: Run tests
+name: Auto Dev Release
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)